src/applications/vaos/appointment-list/components/ScheduleNewAppointmentRadioButtons.jsx

```diff
@@ -75,7 +77,7 @@ function ScheduleNewAppointmentRadioButtons({
 
   return (
     <div className="vads-u-padding-y--3 vads-u-margin-bottom--3 vads-u-border-top--1px vads-u-border-bottom--1px vads-u-border-color--gray-lighter">
-      <h2 className="vads-u-font-size--h3 vads-u-padding-bottom--0 vads-u-margin-y--0">
+      <h2 className="vads-u-padding-bottom--0 vads-u-margin-y--0">
```

Comment:
I'm not sure I understand why the size of this was changed. It looks out of sync with the Express care heading. Also, I think there needs to be more space between the header and the "Choose appointment type" label. The screenshots in the ticket look like it should be about the same amount of space that's between the label and the radio buttons.

![Screen Shot 2021-04-13 at 10 48 55 AM](https://user-images.githubusercontent.com/634932/114572860-e0048080-9c45-11eb-8cce-27cb98af617a.png)

Reply:
@jbalboni, Peter had requested to change the heading size of `Schedule a new appointment` to be in line with `Your appointments` heading.

@outerpress , do you want to change the size of Express Care heading as well?

I do see the need to increase the line spacing between the label and radio button. I'll work on that. 

![image](https://user-images.githubusercontent.com/54327023/114582759-e64b2a80-9c4e-11eb-80d0-e6db8f9bc380.png)

Reply:
@jenniemc let's leave the Express Care heading alone for this one. Thanks!

Reply:
@outerpress @jenniemc Making "Schedule a new appointment" the same size as "Your appointments" on the homepage refresh pages makes it out of sync with the "Your upcoming, past, and Express Care appointments" header on the currently live homepage. I think it needs to look correct on both the refreshed homepage and the current homepage.

Reply:
@jbalboni @outerpress I see a styling difference between the homepage refresh and homepage live (non-refresh). I'll make the changes  to  "Schedule a new appointment"  accordingly to match the heading size  of "Your appointments" on the homepage refresh and the heading size of "Your upcoming, past, and Express Care appointments" on the live homepage (non-refresh).